src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(feature = "nightly", feature(specialization))]
```

Comment:
With the specialization feature removed, isn't there a whole load of stuff in `src/conversion.rs` which breaks?

Reply:
I thought I deleted them...I must've missed some things.